readthedocs/projects/exceptions.py

```diff
@@ -47,6 +47,10 @@ class RepositoryError(BuildEnvironmentError):
         'You can not have two versions with the name latest or stable.'
     )
 
+    FAILED_TO_CHECKOUT = _(
+        'Failed to checkout revision: {}'
```

Comment:
Will this exception log to Sentry? It should not.

Reply:
I don't think so. Actually I'm not sure, I haven't seen a `RepositoryError` in sentry

Reply:
There are some,

https://sentry.io/read-the-docs/readthedocs-org/issues/713027767/?query=RepositoryError

There are some places where it's handled

https://github.com/rtfd/readthedocs.org/blob/d2ea32ae7c25aea609970e598bafa404c0316d38/readthedocs/projects/tasks.py#L252-L273

and there other where it's not: 

https://github.com/rtfd/readthedocs.org/blob/d2ea32ae7c25aea609970e598bafa404c0316d38/readthedocs/projects/tasks.py#L211-L215

I suppose that this is the final solution, https://github.com/rtfd/readthedocs.org/issues/3856

Reply:
Going to merge this, and if it logs, we can handle it later.